apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts

```diff
@@ -37,6 +38,7 @@ export function usePaymentsTableFilters (
     const typeMessage = intl.formatMessage({ id: 'pages.condo.payments.type' })
     const enterTypeMessage = intl.formatMessage({ id: 'pages.condo.payments.enterType' })
     const statusTitle = intl.formatMessage({ id: 'Status' })
+    const paymentOrderTitle = intl.formatMessage({ id: 'PaymentOrder' })
```

Comment:
We use `CamelCaseNames` for translation consts

Reply:
I use CamelCaseNames. Is not it so?

Reply:
**U**pperCamelCase

Reply:
Update const in file